ht2/dz.css

```diff
@@ -0,0 +1,146 @@
+/*переподключить шрифт!!!!!!!!!!!!!*/
+body {
+	font-family: OpenSansRegular;
+	font-size: 13px;
+	background-color: #f9f9f9;	
+}
+/*.....................Сама страница........................*/
+* {
+	padding: 0;
+	margin: 0;
+}
+a {
+	text-decoration: none;
+}
+li {
+	list-style-type: none;
+}
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;
+	background-color: #ffffff;
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*..........................HEADER...........................*/
+/*TOP MENU*/
+.top-menu {
+	display: flex;
+	background-color: #ffffff;
```

Comment:
it is gracefull degradation. If browser is not supporting gradient it will use background-color. But it should be definitely not white but black